packages/lesswrong/components/comments/CommentsNewForm.tsx

```diff
@@ -21,7 +21,7 @@ import { isLWorAF } from '../../lib/instanceSettings';
 import { useTracking } from "../../lib/analyticsEvents";
 import { isFriendlyUI } from '../../themes/forumTheme';
 
-export type CommentFormDisplayMode = "default" | "minimalist"
+export type FormDisplayMode = "default" | "minimalist"
```

Comment:
Now that it's used in another place, maybe this type should live somewhere else? Maybe with the other types in `commentTree.ts`?